tests/storybook.ts

```diff
@@ -4,9 +4,14 @@ import { RunOptions } from '../types'
 export async function test(options: RunOptions) {
 	await runInRepo({
 		...options,
-		repo: 'storybookjs/builder-vite',
-		branch: 'main',
-		build: 'prepublish',
-		test: ['build-examples', 'test-ci']
+		repo: 'storybookjs/repro-templates-temp',
+		dir: 'storybook7',
+		packageDir: 'react-vite/default-js/after-storybook',
+		branch: 'next',
+		build: ['yarn install', 'build-storybook'],
```

Comment:
ecosystem ci calls install, why is it separate here?

Reply:
Maybe it's not needed, I'll try removing it.